scripts/meetup.js

```diff
@@ -8,19 +8,46 @@
 //   None
 //
 // Commands:
-//   hubot when is the next event ?  - returns the next upcoming meetup event
+//   when is the next event ?  - returns the next upcoming meetup event
+//   hubot keep an eye on members - returns when meetup
 
 module.exports = function(robot) {
-    robot.hear(/when(s|'s| is) the next (meetup|event|jsoxford)\s?\?/i, function(msg){
-        var upcomingEventsQuery = "https://api.meetup.com/2/events?offset=0&format=json&limited_events=False&group_id=17778422&only=time%2Cevent_url%2Cname%2Cdescription%2Cyes_rsvp_count%2Crsvp_limit&photo-host=secure&page=20&fields=&order=time&desc=false&status=upcoming&sig_id=153356042&sig=84e9ac6ce37bdb3c00e4f82fe5a7ce798865fbe4";
-        robot.http(upcomingEventsQuery).get()(function(err, res, body){
-          if(err) console.log(err);
-          var upcomingEvents = JSON.parse(body).results;
-          if(upcomingEvents && upcomingEvents.length > 0){
-            msg.send(upcomingEvents[0].event_url);
-          }else{
-            msg.send("No upcoming events planned");
-          }
-        });
+  var membersQuery = "https://api.meetup.com/2/members?offset=0&format=json&group_id=17778422&only=photo%2Cname%2Clink&photo-host=secure&page=200&order=name&sig_id=153356042&sig=4d8e3265b4374b84aabb8efcc26eb8107a3ec81b";
+  var upcomingEventsQuery = "https://api.meetup.com/2/events?offset=0&format=json&limited_events=False&group_id=17778422&only=time%2Cevent_url%2Cname%2Cdescription%2Cyes_rsvp_count%2Crsvp_limit&photo-host=secure&page=20&fields=&order=time&desc=false&status=upcoming&sig_id=153356042&sig=84e9ac6ce37bdb3c00e4f82fe5a7ce798865fbe4";
+  var meetupCheckInterval, result;
+
+  robot.respond(/keep (an eye on|track of) member(s| count)/i, function(msg){
+    if(meetupCheckInterval){
+      msg.send("Already on it!");
```

Comment:
because there is no return from this - it will say "already on it", and then continue to add a new interval anyway.